lang/nb-NO.json

```diff
@@ -46,7 +46,7 @@
   "label.unknown": "Ukjent",
   "label.username": "Brukernavn",
   "label.websites": "Nettsteder",
-  "message.active-users": "{x} nåværende {x, flertall, en {besøkende} annen {besøkende}}",
+  "message.active-users": "{x} nåværende {x, plural, en {besøkende} andre {besøkende}}",
```

Comment:
Check how the Swedish translation is done, should be more like 
"{x} {x, plural, one {besøkende} other {besøkende}} nå"
Hilsen fra Sverige!

Reply:
Tjenare!

Yep, it was fixed in the following commit. 👍 :)

Reply:
Or actually no, see what you mean now. Will push a fix tomorrow. :)